cpp/pmft/PMFTR12.cc

```diff
@@ -130,15 +130,16 @@ void PMFTR12::accumulate(const locality::NeighborQuery* neighbor_query,
             float bin_r = neighbor_bond.distance * dr_inv;
             float bin_t1 = floorf(t1 * dt1_inv);
             float bin_t2 = floorf(t2 * dt2_inv);
-// fast float to int conversion with truncation
+
+            // fast float to int conversion with truncation
 #ifdef __SSE2__
             unsigned int ibin_r = _mm_cvtt_ss2si(_mm_load_ss(&bin_r));
```

Comment:
I'd prefer not to make a change to that in this PR, since it's a simple, self-contained PR, so I'm going to merge without making any changes, but we can continue the discussion here. What are you suggesting that we change?

Reply:
I'm not sure but this isn't going to affect performance much so I'll mark this resolved.